app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -123,8 +123,11 @@ def prepare_for_ep_merge!
     date = Date.strptime(pending_eps.first['date'], '%m/%d/%Y')
     days_ago = (Time.zone.today - date).round
     StatsD.distribution("#{EP_MERGE_STATSD_KEY_PREFIX}.pending_ep_age", days_ago)
-    save_metadata(ep_merge_pending_claim_id: pending_eps.first['id'])
-    add_ep_merge_special_issue! if Flipper.enabled?(:disability_526_ep_merge_api)
+
+    if Flipper.enabled?(:disability_526_ep_merge_api, User.find(user_uuid))
+      save_metadata(ep_merge_pending_claim_id: pending_eps.first['id'])
```

Comment:
Moving `save_metadata()` into this if-enabled section means that later (in `conditionally_merge_ep`) we only need to check for the metadata. The toggle state essentially gets cached by the metadata -- which is unused if the toggle is off anyway.